config/application.properties

```diff
@@ -24,4 +24,8 @@ amqp.messageConverter=io.strimzi.kafka.bridge.amqp.converter.AmqpDefaultMessageC
 #HTTP related settings
 http.enabled=true
 http.host=0.0.0.0
-http.port=8080
\ No newline at end of file
+http.port=8080
+#enable cors
```

Comment:
I am picky ... "Enable CORS", wdyt?

Reply:
done in commit
46d4df9de71e6966a0423ffb719288db793826e0